demo/README.md

```diff
@@ -0,0 +1,140 @@
+# Standalone Rook
```

Comment:
As in a previous comment, most of this readme can be in the main readme. The remaining here would simply be about launching standalone rook and then they could refer back to the same block, object, and file instructions on the main readme.